spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1172,6 +1174,14 @@ public InterpreterResult interpret(String line, InterpreterContext context) {
       return new InterpreterResult(Code.ERROR, "Spark " + sparkVersion.toString()
           + " is not supported");
     }
+
+    ObjectMapper om = new ObjectMapper();
+    om.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+    Map<String, Map<String, String>> userCredentials = om.convertValue(
+            context.getAuthenticationInfo().getUserCredentials().getUserCredentials(), Map.class);
+    interpreter.intp().directBind("userCredentials",
```

Comment:
not necessary - an administrator could have set this up for a team and with this change everyone who has access to the notebook can now see the credentials/password?